check_osp_main.py

```diff
@@ -69,16 +69,14 @@ def main():
         "--warning",
         dest="warning",
         help="Warning value. Could be fraction or whole number.",
-        type=float,
-        default=0.75
```

Comment:
Question: Why not use the default values as floating point? This makes it so that we don't have to check if these are `None` before checking if warning > critical.

Reply:
And will also get rid of that `elif` statement.